app/views/guidance_groups/_guidance_group_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-group">
```

Comment:
nice extraction of the form for being used at new/edit. We should have more of these